oat-sa/extension-tao-sync-server
# test center package

Pull request summary
 
Related to : https://oat-sa.atlassian.net/browse/TAO-9043
 
Full description of changes.
Created data providers and console script which generate test center package
 
#### How to test
Create a test center package using command oat\taoSyncServer\scripts\tools\syncPackage\GeneratePackage an check it for all resourses 

Test commands :

php index.php 'oat\taoSyncServer\scripts\tools\syncPackage\GeneratePackage'

#### Dependencies
    
Requires PR :
https://github.com/oat-sa/extension-tao-sync/pull/171
